tools/ja-rule/JaRuleEndpoint.h

```diff
@@ -45,22 +45,24 @@ class JaRuleEndpoint {
    * @brief The various Ja Rule commands
    */
   typedef enum {
-    ECHO_COMMAND = 0x80,
-    TX_DMX = 0x81,
-    GET_LOG = 0x82,
-    GET_FLAGS = 0x83,
-    WRITE_LOG = 0x84,
-    RESET_DEVICE = 0x85,
-    RDM_DUB = 0x86,
-    RDM_REQUEST = 0x87,
-    SET_BREAK_TIME = 0x88,
-    GET_BREAK_TIME = 0x89,
-    SET_MAB_TIME = 0x90,
-    GET_MAB_TIME = 0x91,
-    SET_RDM_BROADCAST_LISTEN = 0x92,
-    GET_RDM_BROADCAST_LISTEN = 0x93,
-    SET_RDM_WAIT_TIME = 0x94,
-    GET_RDM_WAIT_TIME = 0x95
+    RESET_DEVICE = 0x00,
```

Comment:
Yup, that's coming in a latter change. I should be able to get rid of tools/ja-rule/JaRuleEndpoint.{h,cpp}